docs/docs/Get-Started/get-started-quickstart.md

```diff
@@ -4,74 +4,175 @@ sidebar_position: 2
 slug: /get-started-quickstart
 ---
 
+Get to know Langflow by building an OpenAI-powered chatbot application. After you've constructed a chatbot, add Retrieval Augmented Generation (RAG) to chat with your own data.
 
+## Prerequisites
 
-## Prerequisites {#b5f154a3a1d242c7bdf57acf0a552732}
+* [Python 3.10 to 3.12](https://www.python.org/downloads/release/python-3100/) installed
+* [uv](https://docs.astral.sh/uv/getting-started/installation/), [pip](https://pypi.org/project/pip/), or [pipx](https://pipx.pypa.io/stable/installation/) installed
+* Before installing Langflow, we recommend creating a virtual environment to isolate your Python dependencies with [uv](https://docs.astral.sh/uv/pip/environments), [venv](https://docs.python.org/3/library/venv.html), or [conda](https://anaconda.org/anaconda/conda)
```

Comment:
I think this belongs in the installation topic rather than here. With a quickstart, I think we should assume that the product is already installed.